source/jsonnet/northern-ireland/lib/transforms.libsonnet

```diff
@@ -87,10 +86,22 @@ local listHasSameNameItems = {
   },
 };
 
+local concatenateList = {
```

Comment:
As this isn't a generic `concatenateList` function, this should be named something more specific e.g. `concatenateNames`